src/main/java/strman/Strman.java

```diff
@@ -1334,5 +1336,40 @@ private static long countSubstr(String value, String subStr, boolean allowOverla
         return countSubstr(value.substring(offset), subStr, allowOverlapping, ++count);
     }
 
+    /**
+     * Aggregates the contents of n strings into a single list of tuples.
+     *
+     * @param inputs A list of strings.
+     * @return A list of strings if there are enough elements in the input and none of them is null or empty.
+     * An empty list otherwise.
+     */
+    public static List<String> zip(String... inputs) {
+        if (inputs.length < 2) {
+            return emptyList();
```

Comment:
That does't make any sense.

If input.length is equal to one there is no way we can zip the inputs so the result must be an empty list.

Taking Scala zip function as reference: if you zip two lists and one of them is empty the result will be an empty list. e.g:

``scala> List() zip List(1,2,3)``
``res0: List[(Nothing, Int)] = List()``